pipeline/normalize.py

```diff
@@ -5,7 +5,7 @@
 from collections import Counter
 
 
-USAGE = f"python {__file__} <csv_input_file> <csv_output_file>"
+USAGE = f"python {__file__} <csv_input_file> <csv_output_file> <country>"
```

Comment:
I wonder: Would it be safe to assume that country is US by default? Then you could still run this script without providing the country name as an extra argument for all the current datasets.